content/javascript/concepts/constructors/constructors.md

```diff
@@ -93,7 +93,7 @@ console.log('Car 2 is a ' + car2.year + ' ' + car2.make + ' ' + car2.model);
 // Output: Car 2 is a 2018 Ford Taurus
 ```
 
-### Codebyte Example
+## Codebyte Example
```

Comment:
Hi @mamtawardhani , shouldn't this be a heading level of 3 for proper document structure since it is supplementary content relating to the heading "The Class `constructor` Method" on line 69? Same question for the other heading revision in this commit.

Reply:
To provide further clarification, I think that this is a unique case from the standard set in the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#headerssections) since there are 2 separate sections on this document. Since this is the case, I think it should match the preceding "Example" which is a Heading 3.

Reply:
Checking with this, thanks for the inputs @ryanrackemann 😃